casadm/cas_lib.c

```diff
@@ -2430,6 +2420,9 @@ int partition_set_config(struct kcas_io_classes *cnfg)
 	if (result) {
 		if (OCF_ERR_IO_CLASS_NOT_EXIST == cnfg->ext_err_code) {
 			result = SUCCESS;
+		} else if (OCF_ERR_CACHE_STANDBY == cnfg->ext_err_code) {
+			cas_printf(LOG_ERR, "Cannot set partition config for cache %d while in standby mode.\n", cnfg->cache_id);
```

Comment:
I think `partition` an a name is only used internally - for users it's `IO class config`